packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/ContextLogger.kt

```diff
@@ -80,11 +81,29 @@ public class ContextLogger(public val context: String? = null) {
         doLog(LogLevel.WTF, null, { contextPrefix + message }, *args)
     }
 
+    private inline fun checkPriority(
+        level: LogLevel,
+        block: () -> Unit,
+    ) {
+        if (level.priority >= RealmLog.getLevel(SdkLogCategory).priority) {
```

Comment:
We should probably cache the SDK category level in the SDK, so that we didn't have to round trip core on each  check.